ebean-test/src/test/java/misc/migration/v1_1/ETable.java

```diff
@@ -0,0 +1,49 @@
+package misc.migration.v1_1;
+
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import io.ebean.annotation.DbComment;
+import io.ebean.annotation.History;
+import io.ebean.annotation.Index;
+
+@Table(name = "`table`")
+@Entity
+@History // FIXME: remove later
```

Comment:
@rbygrave It is currently impossible for some platforms to enable/disable history while there are pending drops.
(it will generate the wrong history table)
I don't mind this is a real limitation. I added a sanity check in Ddl-gneration